docs/todo.rst

```diff
@@ -1,6 +1,9 @@
 .. _todo-label:
 
-**To Do**: These are future improvements to Asterix currenly envisionned (not particularly by order of priority):
+Wanna help ?
```

Comment:
How about you convert all of these into GitHub issues (where possible)? In this way all open points and potential improvements are listed in only one place, and they get updated naturally when they have been worked on.

Reply:
Also, it might be more helpful to call this "Open issues" or "Backlog" or something. Sounds a little more serious ^^

Reply:
ahah ok, I'll do it